### ISSUE TITLE
Allow the use of Excel files as input CSV

### ISSUE THREAD

**Issue by mjordan** [mjordan/islandora_workbench#183]
openpyxl can read .xlsx files created by Excel, LibreOffice, and Google Sheets.

**Comment by mjordan** [comment]
This script reads an Excel file, gets the headers and cell values, and writes out a CSV:

```python
#!/usr/bin/env python3

import openpyxl
import csv

wb = openpyxl.load_workbook(filename = 'test.xlsx')
ws = wb['Sheet1']

headers = []
header_row = ws[1]
ws.delete_rows(0)
for header_cell in header_row:
    headers.append(header_cell.value)

records = []
for row in ws:
    record = {}
    for x in range(len(header_row)):
        if headers[x] is not None and row[x] is not None:
            record[headers[x]] =  row[x].value
    records.append(record)

csv_writer_file_handle = open('/tmp/exceltest.csv', 'w+', newline='')
csv_writer = csv.DictWriter(csv_writer_file_handle, fieldnames=headers)
csv_writer.writeheader()
for record in records:
    csv_writer.writerow(record)
csv_writer_file_handle.close()
```

**Comment by mjordan** [comment]
b3e0b535d6afb4552bb582a61e9a6b673ddde6f6 merged work on this so far into main to avoid getting too far out of sync.

**Comment by mjordan** [comment]
Resolved with ff23031d295894e4271b6d1d8ff486b1e2402e6a.
